### PR TITLE
Verification e-mail.

### PR DESCRIPTION
**Pull request by isoos**
#1759

### CODE REVIEW

File: app/lib/shared/email.dart [hunk 1]
```diff
@@ -114,6 +114,31 @@ https://github.com/dart-lang/pub-dartlang-dart/issues
 Pub Site Admin
 ''';
 
-  return new EmailMessage(
-      _defaultFrom, authorizedUploaders, subject, bodyText.toString());
+  return new EmailMessage(_defaultFrom, authorizedUploaders, subject, bodyText);
+}
+
+/// Creates the [EmailMessage] that will be sent to the new uploader for verification.
+EmailMessage createUploaderVerificationEmail({
```
Comment (jonasfj) [inline app/lib/shared/email.dart]: Can we call it a `Confirmation` instead of a `Verification`

File: app/test/shared/email_test.dart [hunk 2]
```diff
@@ -135,4 +135,35 @@ void main() {
...
+        verificationUrl:
+            'https://pub.dartlang.org/verification/add-uploader/abcdef1234567890',
+      );
+      expect(message.from.toString(), 'Pub Site Admin <pub@dartlang.org>');
+      expect(message.recipients.map((e) => e.toString()).toList(),
+          ['uploader@example.com']);
+      expect(message.subject, 'Pub invitation to collaborate: pkg_foo');
+      expect(
+          message.bodyText,
+          'Dear package maintainer,\n'
```
Comment (jonasfj) [inline app/test/shared/email_test.dart]: Do we really want to maintain tests like this?

Wouldn't be easier to just verify that `bodyText` and `subject` contains `pkg_foo` and `abcdef1234567890`, as sanity checks. This other things is just going to break whenever someone changes the email.

Comment (isoos) [reply app/test/shared/email_test.dart]: The point of such tests (both in pana and on the pub site, e.g. golden files) is that we are aware of the extent of the change. It also help us to double-check the results if the templates go more complex.

Comment (sigurdm) [reply app/test/shared/email_test.dart]: I made a small function to maintain goldens in a file: 
https://github.com/dart-lang/dart-protoc-plugin/blob/master/test/golden_file.dart
To update the file, just delete it and run again.

**Pull request merged**